production_services/urls.py

```diff
@@ -28,11 +29,15 @@
     url(r'^admin/', admin.site.urls),
     url(r'^', include('ensembl_bugs.urls')),
     url(r'^$', TemplateView.as_view(template_name='home.html'), name='home'),
+    path('dbcopy/submit/', CopyViews.SubmitView.as_view()),
```

Comment:
I would ask for a proper usrl set, in ensembl_dbcopy/urls.py then included in this main file, such as it's done for ensembl_bugs.

Reply:
Done in latest commit